com.adsbynimbus.nimbus/VersionConstants.cs

```diff
@@ -1,9 +1,10 @@
+
 namespace Nimbus
 {
     public static class VersionConstants
     {
-        public const string UnitySdkVersion = "1.8.1";
         public const string IosSdkVersion = "2.25.5";
-        public const string AndroidSdkVersion = "2.26.1";
+        public const string AndroidSdkVersion = "2.27.0";
+        public const string UnitySdkVersion = "1.9.0";
```

Comment:
Why was Android updated and not iOS? Also if the Unity SDK version is not going to be read dynamically from the package it would be good to update separate from this integration in case this commit had to be reverted which would put the main branch in an inconsistent state.